ios/FluentUI/Vnext/Button/ButtonModifiers.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+public extension MSFButtonView {
+
+    /// Sets the accessibility label for the Button.
+    /// - Parameter accessibilityLabel: Accessibility label string.
+    /// - Returns: The modified Button with the property set.
+    func accessibilityLabel(_ accessibilityLabel: String?) -> MSFButtonView {
+        state.accessibilityLabel = accessibilityLabel
+        return self
+    }
+
+    /// Controls whether the button is available for user interaction, renders the control accordingly.
+    /// - Parameter isDisabled: Boolean value to set the property.
+    /// - Returns: The modified Button with the property set.
+    func isDisabled(_ isDisabled: Bool) -> MSFButtonView {
```

Comment:
Apple has a View extension called `disabled`; can we utilize/override that?

Reply:
We can't override members in extensions in Swift.
We use the disabled modifier in the SwiftUI button in the implementation:

 ```
   public var body: some View {
        Button(action: action, label: {})
            .buttonStyle(MSFButtonViewButtonStyle(tokens: tokens,
                                                  state: state))
            .disabled(state.isDisabled)
            .frame(maxWidth: .infinity)
            .designTokens(tokens,
                          from: theme,
                          with: windowProvider)
    }
```

Reply:
As we spoke offline on this one as well:
 - I'm changing the SwiftUI Button to respect the isEnabled environment value.
 - It was necessary to convert the state property to an optional so we can override it from the UIKit consumption side.
 - A non-optional wrapping property was also needed to be able to comply with the objc bridging which doesn't allow Bool? types.